content/docs/command-reference/status.md

```diff
@@ -65,23 +68,20 @@ stage with differences, the changes in <abbr>dependencies</abbr> and/or
 file name or hash is shown, and additionally a status word is shown describing
 the changes (described below).
 
-- _changed checksum_ means that the stage (in `dvc.yaml`) or `.dvc` file hash
-  has changed (e.g. someone manually edited the file).
-
```

Comment:
Another quick one @efiop, does this status still exist? I removed it for now.

> Unrelated to the PR's original intention I know, sorry.

Reply:
It does still exist :)

Reply:
But the file hashes of .dvc files and dvc.yaml are no longer saved so... what does this check? Maybe i'm confused.

Reply:
@jorgeorpinel It is is still in imported dvc-files. And `dvc add`ed onces will still react to changes in it (backward compatibility). In `dvc.yaml` we do indeed call it differently: "changed command". So I guess we could remove "in dvc.yaml" part here. But I have to point out that we should probably not extend the scope of this PR, it will be painful :slightly_smiling_face:

Reply:
Yeah I will extract this. For now I just removed dvc.yaml from here. Thanks!

Reply:
And extracted to iterative/dvc/issues/4189